### ISSUE TITLE
omitting directory vcl/rules 

### ISSUE THREAD

**Issue by blindpet** [comotion/VSF#28]
Building for Varnish 4.1 and getting this error with make install

```
See any operating system documentation about shared libraries for
more information, such as the ld(1) and ld.so(8) manual pages.
----------------------------------------------------------------------
make[3]: Leaving directory '/root/VSF/libvmod-vsthrottle/src'
make[2]: Leaving directory '/root/VSF/libvmod-vsthrottle/src'
make[2]: Entering directory '/root/VSF/libvmod-vsthrottle'
make[3]: Entering directory '/root/VSF/libvmod-vsthrottle'
make[3]: Nothing to be done for 'install-exec-am'.
 /bin/mkdir -p '/usr/local/share/doc/libvmod-vsthrottle'
 /usr/bin/install -c -m 644 README.rst LICENSE '/usr/local/share/doc/libvmod-vsthrottle'
 /bin/mkdir -p '/usr/local/share/man/man3'
 /usr/bin/install -c -m 644 vmod_vsthrottle.3 '/usr/local/share/man/man3'
make[3]: Leaving directory '/root/VSF/libvmod-vsthrottle'
make[2]: Leaving directory '/root/VSF/libvmod-vsthrottle'
make[1]: Leaving directory '/root/VSF/libvmod-vsthrottle'
install -o root -g root -d /etc/varnish/security
install -o root -g root -D vcl/rules /etc/varnish/security
install: omitting directory ‘vcl/rules’
Makefile:28: recipe for target 'install' failed
make: *** [install] Error 1
```
I can copy them manually but just thought I'd let you know

    cp -r rules /etc/varnish/security/

**Comment by comotion** [comment]
Error message `Makefile:28: recipe for target 'install' failed` indicates that the vmod install failed, can you post the complete log please?

**Comment by blindpet** [comment]
Sure, here you go http://pastebin.com/dE7C6N9g

**Comment by comotion** [comment]
There are some problems with this log. 
Please run again without `-j4` in `make -j4` as it makes the logs hard to follow and can break the build.
There is some problem with the make step as it seems the configure step is never executed for vsthrottle.

**Comment by blindpet** [comment]
Same problem without -j4, http://pastebin.com/nSgFWvcp

**Comment by comotion** [comment]
It seems I was a little optimistic about the install target. Please try again with the new version.

**Comment by blindpet** [comment]
Almost there
```
 /usr/bin/install -c -m 644 README.rst LICENSE '/usr/local/share/doc/libvmod-vst                                                     hrottle'
 /bin/mkdir -p '/usr/local/share/man/man3'
 /usr/bin/install -c -m 644 vmod_vsthrottle.3 '/usr/local/share/man/man3'
make[3]: Leaving directory '/root/VSF/libvmod-vsthrottle'
make[2]: Leaving directory '/root/VSF/libvmod-vsthrottle'
make[1]: Leaving directory '/root/VSF/libvmod-vsthrottle'
install -o root -g root -d /etc/varnish/security
install -o root -g root -d /etc/varnish/security/rules
install -o root -g root -m 644 vcl/vsf.vcl /etc/varnish/security
install -o root -g root -m 644 vcl/config.vcl /etc/varnish/security
install -o root -g root -m 644 vcl/handlers.vcl /etc/varnish/security
install -o root -g root -m 644 vcl/local.vcl.example /etc/varnish/security/local                                                     .vcl
for rule in vcl/rules/cloak.vcl vcl/rules/cmd.vcl vcl/rules/content-encoding.vcl                                                      vcl/rules/content-type.vcl vcl/rules/demo.vcl vcl/rules/dos.vcl vcl/rules/gener                                                     ic.vcl vcl/rules/localfiles.vcl vcl/rules/paths.vcl vcl/rules/php.vcl vcl/rules/                                                     protocol.vcl vcl/rules/request.vcl vcl/rules/restricted-file-extensions.vcl vcl/                                                     rules/robots.vcl vcl/rules/sql.encoded.vcl vcl/rules/sql.vcl vcl/rules/user-agen                                                     t.vcl vcl/rules/xss.encoded.vcl vcl/rules/xss.vcl; do install -o root -g root -m                                                      644 $rule /etc/varnish/security/rules/$rule; done
install: cannot create regular file ‘/etc/varnish/security/rules/vcl/rules/cloak                                                     .vcl’: No such file or directory
install: cannot create regular file ‘/etc/varnish/security/rules/vcl/rules/cmd.v                                                     cl’: No such file or directory
install: cannot create regular file ‘/etc/varnish/security/rules/vcl/rules/conte                                                     nt-encoding.vcl’: No such file or directory
install: cannot create regular file ‘/etc/varnish/security/rules/vcl/rules/conte                                                     nt-type.vcl’: No such file or directory
install: cannot create regular file ‘/etc/varnish/security/rules/vcl/rules/demo.                                                     vcl’: No such file or directory
install: cannot create regular file ‘/etc/varnish/security/rules/vcl/rules/dos.v                                                     cl’: No such file or directory
install: cannot create regular file ‘/etc/varnish/security/rules/vcl/rules/gener                                                     ic.vcl’: No such file or directory
install: cannot create regular file ‘/etc/varnish/security/rules/vcl/rules/local                                                     files.vcl’: No such file or directory
install: cannot create regular file ‘/etc/varnish/security/rules/vcl/rules/paths                                                     .vcl’: No such file or directory
install: cannot create regular file ‘/etc/varnish/security/rules/vcl/rules/php.v                                                     cl’: No such file or directory
install: cannot create regular file ‘/etc/varnish/security/rules/vcl/rules/proto                                                     col.vcl’: No such file or directory
install: cannot create regular file ‘/etc/varnish/security/rules/vcl/rules/reque                                                     st.vcl’: No such file or directory
install: cannot create regular file ‘/etc/varnish/security/rules/vcl/rules/restr                                                     icted-file-extensions.vcl’: No such file or directory
install: cannot create regular file ‘/etc/varnish/security/rules/vcl/rules/robot                                                     s.vcl’: No such file or directory
install: cannot create regular file ‘/etc/varnish/security/rules/vcl/rules/sql.e                                                     ncoded.vcl’: No such file or directory
install: cannot create regular file ‘/etc/varnish/security/rules/vcl/rules/sql.v                                                     cl’: No such file or directory
install: cannot create regular file ‘/etc/varnish/security/rules/vcl/rules/user-                                                     agent.vcl’: No such file or directory
install: cannot create regular file ‘/etc/varnish/security/rules/vcl/rules/xss.e                                                     ncoded.vcl’: No such file or directory
install: cannot create regular file ‘/etc/varnish/security/rules/vcl/rules/xss.v                                                     cl’: No such file or directory
Makefile:30: recipe for target 'install' failed
make: *** [install] Error 1
```

**Comment by comotion** [comment]
yep. didn't test it well enough.
